Make pkg/util golint compatible

danielcooperxyz opened the pull request:
<!-- Thank you for contributing!

     Read more about how you can contribute in our contribution guide:
     https://github.com/kedacore/keda/blob/master/CONTRIBUTING.md
-->

_Provide a description of what has been changed_
Make pkg/util golint compatible

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Fixes #1058

zroubalik commented:
IMHO, you should update the exclude section in https://github.com/kedacore/keda/blob/v2/.pre-commit-config.yaml#L9

turbaszek commented:
> IMHO, you should update the exclude section in https://github.com/kedacore/keda/blob/v2/.pre-commit-config.yaml#L9
> 
> @turbaszek am I right?

Yup, in this way we will start checking those files on CI

zroubalik commented on version/version.go:
```diff
@@ -1,6 +1,8 @@
 var (
-	Version   = "v2"
+	// Version defines the current version of keda
+	Version = "v2"
```
`sed` command in the Makefile needs to be probably updated because of this change:
https://github.com/kedacore/keda/blob/598f1a4985fe4965a579695f195701472ae079ea/Makefile#L81

danielcooperxyz replied:
Thanks for the feedback! I tested this as it is and it works as it should. I think thats because the pattern used matches the full assignment statement, not just the variable name.

To test I used the below in the Makefile

```
.PHONY: get-version
get-version:
	
	@sed -i 's@Version   =.*@Version   = "$(VERSION)"@g' ./version/version.go;
	@echo $(VERSION);
```

danielcooperxyz commented:
> 
> 
> > IMHO, you should update the exclude section in https://github.com/kedacore/keda/blob/v2/.pre-commit-config.yaml#L9
> > @turbaszek am I right?
> 
> Yup, in this way we will start checking those files on CI. Also the issue mentions ` pkg/util/normalize_string.go`, is this file golint compatible @danielcooperxyz?

Thanks for the feedback! I've updated the .pre-commit-config.yaml and updated the normalise_string.go file, this must have been lost when I replayed my commits to include signoffs, sorry!

turbaszek reviewed (approved):
Nice job @danielcooperxyz 🚀

zroubalik reviewed (approved):
LGTM, thanks @danielcooperxyz!
